File: Program83.java
import java.lang.*;
import java.util.*;

class Pattern
{
    private int iNo;

    public Pattern(int iValue)
    {
        this.iNo = iValue;
    }
    public void Display()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
             if((iCnt % 2) == 0)
             {
                 System.out.print("*\t");
             }
             else
             {
                 System.out.print(iCnt+"\t");
             }
        }
    } 
}

class Program83
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        System.out.println("Enter number");
        iValue = sobj.nextInt();

        Pattern obj = new Pattern(iValue);

        obj.Display();
    }
}

File: Program83.cpp
#include<iostream>
using namespace std;

class Pattern
{
    private:
       int iNo;
    public:
        Pattern(int iValue)
        {
            this->iNo = iValue;
        }
        void Display()
        {
            int iCnt = 0;

            for(iCnt = 1; iCnt <= iNo; iCnt++)
            {
                if((iCnt % 2) == 0)
                {
                    cout<<"*\t";
                }
                else
                {
                    cout<<iCnt<<"\t";
                }
            }
        }
};

int main()
{
    int iValue = 0;

    cout<<"Enter number"<<endl;
    cin>>iValue;

    Pattern obj(iValue);

    obj.Display();
}